crtclient/docker/docker.go

```diff
@@ -75,7 +75,7 @@ func (d *Docker) getContPid(ctx *crtclient.ContainerEpContext) (string, error) {
 	contInfo, err := d.Client.InspectContainer(contNameOrId)
 	if err != nil {
 
-		log.Printf("unable to get container info for '%s' \n",
+		log.Errorf("unable to get container info for '%s' \n",
```

Comment:
Just pinging folks - is it a good idea to come up with a standard format of an error and warning messages? It wouldn't be part of this checkin regardless.